content/blog/cloud-engineering-summit/index.md

```diff
@@ -0,0 +1,247 @@
+        <ul style="list-style-type:none;">
+            <li><b>Maya Kaczorowski</b>, Senior Director, Product Management, Software Supply Chain Security, GitHub</li>
+            <li><b>Amit Gupta</b>, VP Business Development and Product Management, Tigera</li>
+            <li><b>Johnathon Hunt</b>, VP of Security, GitHub</li>
+            <li>Moderator: Eric Rudder</li>
+        </ul>
+    </div>
+</div>
+
+<div style="clear:both;padding-top:20px;">
+    <div style="float:left;border-width:2px;">
+        <div style="display:inline-block;vertical-align:top;">
+            <div style="float:left;">
+                <img src="abby-kearns.jpeg" alt="img" style="float:left;width:100px;height:100px;"/>
+            </div>
+            <div style="padding-top:105px;">
+                <img src="johnathon-sullivan.png" alt="img" style="width:100px;height:100px;"/>
+            </div>
+        </div>
+        <div style="display:inline-block;vertical-align:top;">
+            <div style="float:left;">
+                <img src="amanda-silver.jpg" alt="img" style="width:100px;height:100px;"/>
+            </div>
+            <div style="padding-top:105px;">
+                <img src="avi-cavale.jpeg" alt="img" style="width:100px;height:100px;"/>
+            </div>
+        </div>
+    </div>
+    <div style="padding-left:225px;vertical-align:bottom;">
+        <b>Future of Cloud Engineering: Culture, Process, Tools</b>
+        <ul style="list-style-type:none;">
+            <li><b>Abby Kearns</b>, CTO, Puppet/li>
+            <li><b>Amanda Silver</b>, CVP of Product for Developer Tools, Microsoft</li>
+            <li><b>Johnathon Sullivan</b>, Co-founder and CTO, NS1</li>
+            <li><b>Avi Cavale</b>, VP of Engineering, JFrog</li>
+            <li>Moderator: Lee Zen</li>
+        </ul>
+    </div>
+</div>
+
+<div style="display:inline-block;vertical-align:top;padding-top:18px;">
+    <span>
+        <img src="nenne-adaora-nwodo.jpg" alt="img" style="float:left;display:inline;clear:both;width:100px;height:100px;postion:absolute;border-width:2px;"/>
+        <div style="padding-left:120px;padding-top:5px;padding-bottom:45px">
+            <b>Managing Your Cloud Application and Infrastructure Deployments in One CI/CD Pipeline</b>
+            <ul style="list-style-type:none;">
+                <li>Nenne Adaora (Adora) Nwodo, Software Engineer, Microsoft</li>
+            </ul>
+        </div>
+    </span>
+</div>
+
+<div style="display:inline-block;vertical-align:top;">
+    <span>
+        <img src="charity-majors.jpg" alt="img" style="float:left;display:inline;clear:both;width:100px;height:100px;postion:absolute;border-width:2px;"/>
+        <div style="padding-left:120px;padding-top:5px;padding-bottom:70px;">
+            <b>Testing in Production</b>
+            <ul style="list-style-type:none;">
+                <li>Charity Majors CTO and Co-founder, Honeycomb</li>
+            </ul>
+        </div>
+    </span>
+</div>
+
```

Comment:
Charity has not responded to email -- we should hold on publishing her name until we know if she is actually going to speak.